### PR TITLE
Add bulk array instructions to MVP.md

### PR DESCRIPTION
**Pull request by tlively**
Add descriptions of array.fill, array.copy, array.init_elem, and
array.init_data to MVP.md.

### CODE REVIEW

**Comment by tlively** [comment]
Current dependencies on/for this PR:
* main
  * **PR #375** <a href="https://app.graphite.dev/github/pr/WebAssembly/gc/375" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/WebAssembly/gc/375?utm_source=stack-comment).

File: proposals/gc/MVP.md [hunk 1]
```diff
@@ -576,6 +576,55 @@ In particular, `ref.null` is typed as before, despite the introduction of `none`
...
+
+* `array.init_data` copies a sequence of values from a data segment to an array
+  - `array.init_data $t $d : [(ref null $t), i32, i32, i32 ] -> []`
+    - iff `expand($t) = array (mut t)`
+    - and `t` is numeric, vector, or packed
+    - and `$d` is a defined data segment
+  - the 1st operand is the `array` to be initialized
+  - the 2nd operand is the `dest_offset` at which the copy will begin in `array`
+  - the 3rd operand is the `src_offset` at which the copy will begin in `$d`
+  - the 4th operand is the `size` of the copy in array slots
```
Comment (rossberg) [inline proposals/gc/MVP.md:625]: This probably ought to explain that for a packed field type `t`, the source is considered to be packed in the same way.

Comment (rossberg) [inline proposals/gc/MVP.md:625]: This probably ought to explain that for a packed field type `t`, the source is considered to be packed in the same way.

File: proposals/gc/MVP.md [hunk 2]
```diff
@@ -576,6 +576,55 @@ In particular, `ref.null` is typed as before, despite the introduction of `none`
...
+  - traps if `array` is null
+  - traps if `dest_offset + size > len(array)` or `src_offset + size * bytes(t) > len($d)`
```
Comment (rossberg) [inline proposals/gc/MVP.md:627]:
```suggestion
  - traps if `dest_offset + size > len(array)` or `src_offset + size * |t| > len($d)`
```

Comment (rossberg) [inline proposals/gc/MVP.md:627]:
```suggestion
  - traps if `dest_offset + size > len(array)` or `src_offset + size * |t| > len($d)`
```

**Review by rossberg — APPROVED** [review]
LGTM modulo comments

File: proposals/gc/MVP.md [hunk 3]
```diff
@@ -576,6 +576,55 @@ In particular, `ref.null` is typed as before, despite the introduction of `none`
...
 
+* `array.fill` fills a slice of an array with a given value
+  - `array.fill $t : [(ref null $t), i32, t, i32] -> []`
+    - iff `expand($t) = array (mut t')`
+    - and `t = unpacked(t')`
+  - the 1st operand is the `array` to fill
+  - the 2nd operand is the `offset` into the array at which to begin filling
+  - the 3rd operand is the `value` with which to fill
+  - the 4th operand is the `size` of the filled slice
+  - traps if `array` is null or `offset + size > len(array)`
```
Comment (Liedtke) [inline proposals/gc/MVP.md:587]: It should also trap if the offset or size are negative afaik. (Similar comment for the operations below.)

**Comment by tlively** [comment]
All comments addressed, so merging now. Happy to continue discussions and make follow-up PRs if necessary.

**Pull request merged**